lib/archethic/transaction_chain.ex

```diff
@@ -1013,7 +1013,11 @@ defmodule Archethic.TransactionChain do
   @spec fetch_genesis_address_remotely(address :: binary(), list(Node.t())) ::
           {:ok, binary()} | {:error, :network_issue}
   def fetch_genesis_address_remotely(address, nodes) when is_binary(address) do
-    case P2P.quorum_read(nodes, %GetGenesisAddress{address: address}) do
+    conflict_resolver = fn results ->
+      Enum.min_by(results, & &1.timestamp)
```

Comment:
You should specify the sorter function for a DateTime like `Enum.min_by(results, & &1.timestamp, DateTime)` as it is described in DateTime doc